package.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "vscode-cosmosdb",
     "version": "0.19.3-alpha.0",
-    "aiKey": "0c6ae279ed8443289764825290e4f9e2-1a736e7c-1324-4338-be46-fc2a58ae4d14-7255",
```

Comment:
This needs to remain at the value currently in the code.

Reply:
Got it. 

Just so I know, why did we change it for Functions? I thought it had to do with upgrading the telemetry package.